tests/foreman/cli/test_reporttemplates.py

```diff
@@ -654,3 +655,41 @@ def test_negative_nonauthor_of_report_cant_download_it(self):
                 'id': report_template['name'],
                 'job-id': schedule[0].split("Job ID: ", 1)[1],
             })
+
+    @skip_if_bug_open('bugzilla', 1750924)
```

Comment:
now that https://github.com/SatelliteQE/robottelo/pull/7404 is merged we'll probably need  @pytest.mark.skip_if_open('BZ:1750924') here